client/mm/config.go

```diff
@@ -63,6 +74,10 @@ func (c *BotConfig) requiresPriceOracle() bool {
 	return false
 }
 
+func (c *BotConfig) requiresCEX() bool {
+	return c.SimpleArbConfig != nil || c.MMWithCEXConfig != nil
```

Comment:
Why not just make `BotCEXCfg` a filed of, or embedded in, `SimpleArbConfig` and `MMWithCEXConfig`?

Reply:
I did it like this because the strategy doesn't need to know anything about the CEX it's trading on. Creating the CEXes  and allocating the balances is the purview of the level above the strategies. I was kind of split on this, thought it could go either way.